examples/calculator.py

```diff
@@ -28,7 +28,8 @@ class CalculatorApp(App):
         "plus_minus_sign": "plus-minus",
         "percent_sign": "percent",
         "equals_sign": "equals",
-        "enter": "equals",
```

Comment:
Presumably the intention here is to allow full keyboard-only button navigation, which then requires that `Enter` presses the current button rather than act as an alternative for `=`? That's fair.